apps/api/app.js

```diff
@@ -4,15 +4,44 @@ import express from "express";
 import router from "./src/router";
 import bodyParser from "body-parser";
 import cors from "cors";
+import passport from "passport";
+import { ExtractJwt, Strategy } from "passport-jwt";
+
 import db from "./src/models";
 import Location from "./src/models/Location";
+import User from "./src/models/User";
+
 import { locationsSeed } from "./dbSeed";
 
 const app = express();
 const PORT = process.env.PORT || 3001;
 
-app.use(bodyParser.json());
+//Config JwtStrategy
+const JWT_STRATEGY_CONFIG = {
+  jwtFromRequest: ExtractJwt.fromExtractors([
+    ExtractJwt.fromAuthHeaderAsBearerToken(),
+    ExtractJwt.fromBodyField("auth_token"),
+    ExtractJwt.fromUrlQueryParameter("auth_token"),
+  ]),
+  secretOrKey: process.env.JWT_SECRET,
+  algorithms: [process.env.JWT_ALGORITHM],
+};
+passport.use(
+  new Strategy(JWT_STRATEGY_CONFIG, async (jwtPayload, done) => {
+    try {
+      const user = await User.findById(jwtPayload._id);
 
+      if (user) return done(null, user);
+      else return done(null, false);
```

Comment:
I think the `else` is not necessary
```suggestion
      if (user) return done(null, user);
      
      return done(null, false);
```